packages/inter-protocol/src/psm/psm.js

```diff
@@ -105,10 +106,9 @@ export const prepare = async (zcf, privateArgs, baggage) => {
   const { anchorBrand, anchorPerMinted } = zcf.getTerms();
   console.log('PSM Starting', anchorBrand, anchorPerMinted);
 
-  const stableMint = await zcf.registerFeeMint(
-    'Minted',
-    privateArgs.feeMintAccess,
-  );
+  const { stableMint } = await provideAll(baggage, {
+    stableMint: () => zcf.registerFeeMint('Minted', privateArgs.feeMintAccess),
```

Comment:
seems a little odd that we have to preserve the identity of the stable mint on our end; seems like calling `registerFeeMint` in each incarnation should be fine.

I suppose what we need to do is avoid registering the keyword `Minted` twice.

Reply:
yup